Components/login.css

```diff
@@ -1,14 +1,15 @@
 .login-container{
     background-color: var(--color-primary);
     width: auto;
-    height: 35rem;
+    height: 30rem;
 }
 .input-form{
-    width: 30rem;
+    width: 27rem;
     border-radius: 1rem;
     height: 30rem;
     align-self: center;
     margin-top: 2rem;
```

Comment:
use shorthand property to define margin, padding i.e Top, Right, Bottom, Left

Reply:
Thanks, done.